guides/request-validation-python/example-2/example-2.5.x.py

```diff
@@ -1,9 +1,10 @@
 from flask import Flask, request
 from twilio import twiml
-
+from . import validate_twilio_request
```

Comment:
Relative import in Python works only inside packages (directories containing a `__init__.py` file). For snippets we shouldn't rely on any other local files.